pi pico net examples (#832)
ZigEmbeddedGroup/microzig: examples/raspberrypi/rp2xxx/src/net/readme.md

# Examples for the pico 2w and lwip network

Currently tested only on pico 2w, should also work on pico w.

## secrets.zig

Enter your ssid/pwd into `secrets.zig` file. All examples will used that credentials to connect to the wifi. There is option to use dhcp or fixed ip address. 

> `tcp_client` example also uses `host_ip` when connecting to the host, enter there your desktop host ip address.

Reviewer: ```suggestion
`tcp_client` example also uses `host_ip` when connecting to the host, enter your desktop host IP address there.
```

Joining wifi defaults to the `wpa2_psk` wifi security and country code "XX" (worldwide), if that does not match your configuration add options to the wifi.join:

```zig
try wifi.join(
    secrets.ssid,
    secrets.pwd,
    .{ .security = .open, .country = .{ .code = "US".* } },
);
```

## pong.zig

> This only setups cyw43 (pico wifi chip), joins wifi network and initializes lwip stack.   

Reviewer: ```suggestion
This only sets up cyw43 (pico wifi chip), joins the WiFi network, and initializes lwip stack.   
```

> When connected it will display ip address in the log:

Reviewer: ```suggestion
When connected it will display its IP address in the log:
```
